### DISCUSSION TITLE
演示站日历部分

### DISCUSSION THREAD

**Discussion by sbcxk** [solstice23/argon-theme#218] [Q&A]
![3L506CPI0C2`3$I}N3CA}$1](https://user-images.githubusercontent.com/67060479/120887157-dbe05800-c623-11eb-9345-ba1631115dd7.png)
想请教一下 演示站的这种日历部分和年度进程怎么设置呀，自己的日历太丑了
![K3_5 T4 {$W}Y5C 4{6 ~GI](https://user-images.githubusercontent.com/67060479/120887196-10541400-c624-11eb-8646-2f0a10f1c023.png)

**Comment by yzxoi** [comment]
wp 的小工具内插入自定义 HTML，放上单向历

**Reply by sbcxk** [reply]
好的 单向历是需要自己写或者网上找吧
